src/AutoRest.CSharp.V3/Output/Models/ResourceOperation.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using AutoRest.CSharp.V3.Input;
+using AutoRest.CSharp.V3.Output.Builders;
+using AutoRest.CSharp.V3.Output.Models.Types;
+using AutoRest.CSharp.V3.Utilities;
+
+namespace AutoRest.CSharp.V3.Output.Models
+{
+    internal class ResourceOperation : TypeProvider
+    {
+        private const string ClientSuffixValue = "Client";
+        private const string OperationsSuffixValue = "Operations";
+        private OperationGroup _operationGroup;
+
+        public ResourceOperation(OperationGroup operationGroup, BuildContext context)
+            : base(context)
+        {
+            _operationGroup = operationGroup;
+            var clientPrefix = GetClientPrefix(operationGroup.Language.Default.Name);
```

Comment:
```suggestion
            string clientPrefix = GetClientPrefix(operationGroup.Language.Default.Name);
```

No var when right hand doesn't show type.